### PR TITLE
replace old \blue \green \red with the <span>s also emojis

### PR DESCRIPTION
**Pull request by spookydonut**
:cl: Ondrej008
:add: Emojis, once GoonChat is here.
/:cl:

Resolves conflicts with #292 

closes #292

### CODE REVIEW

File: code/ATMOSPHERICS/pipes.dm [hunk 1]
```diff
@@ -199,7 +199,7 @@
 	return 1
 
 /obj/machinery/atmospherics/pipe/simple/proc/burst()
-	src.visible_message("\red \bold [src] bursts!");
+	src.visible_message("<span class='warning'> \bold [src] bursts!</span>");
```
Comment (DominikPanic) [inline code/ATMOSPHERICS/pipes.dm]: Bold!

File: code/modules/admin/NewBan.dm [hunk 2]
```diff
@@ -168,7 +168,7 @@ var/savefile/Banlist
 /datum/admins/proc/unbanpanel()
 	var/count = 0
 	var/dat
-	//var/dat = "<HR><B>Unban Player:</B> \blue(U) = Unban , (E) = Edit Ban\green (Total<HR><table border=1 rules=all frame=void cellspacing=0 cellpadding=3 >"
+	//var/dat = "<HR><B>Unban Player:</B> <span class='notice'>(U) = Unban , (E) = Edit Ban<span class='green'> (Total<HR><table border=1 rules=all frame=void cellspacing=0 cellpadding=3 ></span>"
```
[the remark this answers is not in the document]
Comment (spookydonut) [reply code/modules/admin/NewBan.dm]: the admin stuff is all kinds of fucky

[the remark this answers is not in the document]
Comment (spookydonut) [reply code/modules/admin/NewBan.dm]: pls fix when you port new admin tools

[the remark this answers is not in the document]
Comment (DominikPanic) [reply code/modules/admin/NewBan.dm]: I will spookster, I will.

File: code/game/objects/structures/plants.dm [hunk 3]
```diff
@@ -1,172 +1,172 @@
...
-
-		overlays -= fruit_overlay
-		fruit_overlay = icon('icons/obj/structures/jungle.dmi',"fruit[fruits_left]")
-		fruit_overlay.Blend(rgb(fruit_r, fruit_g, fruit_b), ICON_ADD)
-		overlays += fruit_overlay
-	else
-		to_chat(user, "\red There are no fruit left on [src].")
-		
\ No newline at end of file
+//*********************//
```
[the remark this answers is not in the document]
Comment (spookydonut) [reply code/game/objects/structures/plants.dm]: i think its just line endings.

**Pull request merged**